lib/pure/math.nim

```diff
@@ -1131,3 +1133,24 @@ func lcm*[T](x: openArray[T]): T {.since: (1, 1).} =
   while i < x.len:
     result = lcm(result, x[i])
     inc(i)
+
+
+when not defined(js) and not defined(nimscript):
+  func copySign*[T: SomeFloat](x, y: T): T {.inline, since: (1, 5, 1).} =
+    ## Returns a value with the magnitude of `x` and the sign of `y`.
+    runnableExamples:
+      doAssert copysign(10.0, -1.0) == -10.0
```

Comment:
style: `copySign`

(should we start using `--styleCheck:error` for running runnableExamples in stdlib?)